Magento/Sniffs/PHP/DiscouragedFunctionSniff.php

```diff
@@ -235,7 +235,7 @@ class DiscouragedFunctionSniff extends ForbiddenFunctionsSniff
      *
      * @return void
      */
-    protected function addError($phpcsFile, $stackPtr, $function, $pattern = null)
+    protected function addError($phpcsFile, $stackPtr, $function, $pattern=null)
```

Comment:
Where such spaces removal around `=` comes from?

Reply:
You're right. I missed that part of PSR2 standard. Will fix.

Reply:
I fixed some rules to be compatible with PSR2. 
Anyway, severities are all about Marketplace checks so you don't have to worry about Magento core. For core all findings will lead to build failure.

Reply:
@lenaorobei I wouldn't worry about Magento core is there was a separate Marketplace coding standard based on core one. What is the reason to mix such totally different concerns?

Reply:
@orlangur why do you think so? Extension developers should follow the same rules as core developers.